### PR TITLE
url youtube added media response

### PR DESCRIPTION
**Pull request by AliceYuan**
forgot to add this in previous commit

### CODE REVIEW

File: client/scraper/youtube_metadata.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def parse_content(self, response):
         media_list = {}
         media_list[FieldKeyword.DATA] = [
             {
-                FieldKeyword.URL: None,
+                FieldKeyword.URL: response.request_url,
```
Comment (MeenuV) [inline client/scraper/youtube_metadata.py]: What's the reasoning behind this living inside the media_list rather than outside as part of the prop_map?

Comment (AliceYuan) [inline client/scraper/youtube_metadata.py]: uh not sure, I could make it something like... api_request_url
instead

Comment (MeenuV) [inline client/scraper/youtube_metadata.py]: Does this refer to the oembed link? Or the regular sanitized url? Cause if it's the latter, it should just live in the prop_map no?

Comment (AliceYuan) [inline client/scraper/youtube_metadata.py]: oembed link

Comment (MeenuV) [inline client/scraper/youtube_metadata.py]: Will we run into a case where we might be nesting multiple youtube requests into one response? I feel like this url should live in prop_map along with all other request/sanitized urls. Cause there's only one oembed link that maps to a particular youtube url

**Comment by AliceYuan** [comment]
closing, see: https://github.com/SAAVY/magpie/pull/72

**Pull request closed without merging**